### DISCUSSION TITLE
Why does he reset every once in a while

### DISCUSSION THREAD

**Discussion by shijunti19** [TabbyML/tabby#2870] [Q&A] [answered]
I didn't pay, I used the free version

https://tabby.tabbyml.com/docs/quick-start/installation/modal/

**modal deploy app.py** 

Why does the mod website reset every once in a while after deployment, if it is allowed to run permanently and save files

![image](https://github.com/user-attachments/assets/16659eb1-b666-45b8-a244-1249900523e0)

How should I deploy it to work properly？

![image](https://github.com/user-attachments/assets/5c97ddc2-9bd7-47ca-8cbc-5a008bca7500)
I have already set it up, but it has been reset again

**Comment by wsxiaoys** [comment, marked as answer]
Hello -By default, the modal uses ephemeral storage. To utilize persistent storage, you'll need to employ its persistent disk feature.

This was briefly discussed in https://github.com/TabbyML/tabby/pull/2593/files, but it has not been merged yet.

**Comment by shijunti19** [comment]
Okay, I'm waiting for your release. I also noticed this issue myself, but after trying, I couldn't do it myself.
